### PR TITLE
WIP: Filemanger app

### PR DESCRIPTION
**Pull request by janekx21**
# #57  Filemanager App

## Description
This is a very crude info implamentation. It just shows the files in a list. Interaction with the files is also not needet, because there are not that many programs that open files (rilly none).

## Related Issue
#57 

## Motivation and Context
The filemanager app is rilly a base for the programs that open files later.

## Screenshots (if appropriate)
(add if you like)

## Type of change

- [ ] Bug fix
- [x] Feature request
- [ ] Documenting code

## Checklists

- [x] I referenced the according Issue
  - [x] I created an Issue
  - [x] I put a link in the Section [Related Issue](#related-issue)
- [ ] I tested my code with multiple browsers
- [x] I followed the [Angular style guide](https://angular.io/guide/styleguide)
- [x] My commit messages are good and understandable

### Optional

- [x] I linted my code with TSLint
- [ ] I tested my code with Karma
- [ ] I tested my code with Protector

### CODE REVIEW

File: src/app/desktop/windows/file-manager/file-manager.component.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<div id="file-manager-window-container">
+  <div id="file-manager-window">
+    <div class="category">
+      <h2>Data</h2>
+      <div class="hr" style="width: 100%"></div>
```
Comment (use-to) [inline src/app/desktop/windows/file-manager/file-manager.component.html]: Do not inline style!

File: src/app/desktop/windows/file-manager/file-manager.component.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  }
+
+  ngOnInit() {
+    this.update();
+
+    // update shown files every 1 sec
+    setInterval(() => this.update(), 1000);
+  }
+
+  private update(): void {
```
Comment (use-to) [inline src/app/desktop/windows/file-manager/file-manager.component.ts]: Tests?

File: src/app/desktop/windows/file-manager/file-manager.component.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+  constructor(private websocket: WebsocketService) {
+    super();
+  }
+
+  ngOnInit() {
+    this.update();
+
+    // update shown files every 1 sec
+    setInterval(() => this.update(), 1000);
```
Comment (use-to) [inline src/app/desktop/windows/file-manager/file-manager.component.ts]: So what?! You shouldnt send the same request every second.
Change this!

Comment (MaxiHuHe04) [reply src/app/desktop/windows/file-manager/file-manager.component.ts]: Maybe we should get a notification when a file is changed/created

**Pull request closed without merging**